docs/zh_cn/quick_start_guide.md

```diff
@@ -0,0 +1,206 @@
+# JuiceFS 快速上手
+
+[TOC]
```

Comment:
[TOC] markup  is not work

Reply:
It seems [TOC] markup is not supported by Github. I will try some tools for generating TOC code for a single page.